src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html

```diff
@@ -19,6 +19,11 @@ <h2 class="h2 {% if userfeed.action_required %}indicator{% endif %}">
         <a href="{{ item.action_url }}" class="card {% if item.action_required %}card--status{% endif %} {% if item.is_completed %}card--completed{% endif %}">
             <div class="card__body card__body--tabled">
                 <p class="p tabled__value">{{ item.title }}</p>
+                                
+                <p class="p" style="color: rebeccapurple; font-size: 300%;">
+                    {{ item.status_indicator }}
+                </p>
+                
```

Comment:
@jiromaykin `item.status_indicator` is one of `open_inwoner.openzaak.constants.StatusIndicators`